src/getTime.js

```diff
@@ -20,11 +20,7 @@
 function getTime(str) {
   const time = str.match(/(\D|^)((([2][0-3])|([0-1][0-9])):[0-5][0-9])(\D|$)/);
```

Comment:
`(\D|^)` matches any non-numeric symbol before and after HH:MM. It's better to use `\b`